navigation/src/main/java/com/graphhopper/navigation/NavigateResponseConverter.java

```diff
@@ -205,29 +226,29 @@ private static void putLegInformation(ObjectNode legJson, ResponsePath path, int
      * The first Intersection of the first step should only have one "bearings" and one
      * "out" entry
      */
-    private static void fixFirstIntersectionDetail(List<PathDetail> intersectionDetails) {
+    private static void fixDepartIntersectionDetail(List<PathDetail> intersectionDetails, int pos) {
```

Comment:
I don't like the name pos, can you rename this to something that explains what pos is?